mage_ai/data_preparation/models/block.py

```diff
@@ -149,10 +150,10 @@ def delete(self):
             p.delete_block(p.get_block(self.uuid))
         os.remove(self.file_path)
 
-    async def execute(self, analyze_outputs=True, custom_code=None):
+    async def execute(self, analyze_outputs=True, custom_code=None, redirect_outputs=False):
         with VerboseFunctionExec(f'Executing {self.type} block: {self.uuid}'):
             try:
-                output = await self.execute_block(custom_code)
+                output = await self.execute_block(custom_code, redirect_outputs)
```

Comment:
nit: let's write `custom_code=custom_code, redirect_outputs=redirect_outputs` for keyword arguments.

Reply:
Fixed.